packages/sommelier.rb

```diff
@@ -60,6 +60,7 @@ def self.build
       system "echo 'SOMM=\$(pidof sommelier 2> /dev/null)' >> stopsommelier"
       system "echo 'if [ ! -z \"\$SOMM\" ]; then' >> stopsommelier"
       system "echo '  killall -g sommelier' >> stopsommelier"
+      system "echo '  pkill Xwayland' >> stopsommelier"
```

Comment:
Should we test if Xwayland is running first before attempting to kill?  Otherwise, it will throw an error.

Reply:
I thought about adding a `restartsommelier` command also.  Basically would just run `stopsommelier && startsommelier`.

Reply:
this should work
```sh
if ! pgrep "xwayland"; then killall Xwayland; fi
```

Reply:
Changed to if then format as per @ThatGeekyWeeb suggestion.